style2.css

```diff
@@ -0,0 +1,716 @@
+  }
+
+  .para2 {
+    margin: 1em 187px 2em 1em;
+  }
+
+  .para1 {
+    margin-top: 1em;
+    margin-bottom: 2em;
+    background-color: white;
+
+    --bs-gutter-y: 0;
+
+    padding-left: 20px;
+    padding-right: 20px;
+    margin-left: 2em;
+    margin-right: 187px;
+    border: 2px solid #8080804d;
+    font-size: 24px;
+  }
+
+  .para1 p {
+    margin: 1.0625rem 0.75em 0;
+    width: calc(100% - 2.5em);
+    padding: 1em 0.5em;
+    line-height: 150%;
+    font-weight: normal;
+    font-size: 0.8125rem;
+    font-family: var(--ff-lato);
+    height: auto;
+    text-align: justify;
+    border: 0.25rem solid var(--clr-neutral-white);
+  }
+
+  .second {
+    padding-left: 14em;
+  }
+
+  .para2-first {
+    padding-left: 3em;
+    line-height: 1.375rem;
+  }
+
+  .light {
+    font-family: 'Lato', sans-serif;
+    font-size: 18px;
+    color: #6f7073;
+  }
+
+  .image {
+    width: 31em;
+    background-color: white;
+
+    --bs-gutter-y: 0;
+
+    padding: 30px;
+    margin-left: 5em;
+    border: 2px solid #8080804d;
+  }
+
+  .past-summit {
+    padding-top: 1em;
+    flex-direction: column;
+    gap: 2em;
+  }
+
+  .str {
+    font-size: 37px;
+    width: 15em;
+    margin-left: 4em;
+  }
+
+  .lig {
+    font-family: 'Lato', sans-serif;
+    font-size: 18px;
+    color: #6f7073;
+    margin-left: 4em;
+    padding-left: 85px;
+  }
+
+  .cut {
+    font-family: 'Lato', sans-serif;
+    font-size: 18px;
+    color: #6f7073;
+    padding-left: 191px;
+  }
+
+  .history {
+    display: flex;
+    gap: 1em;
+  }
+
+  .first-histo {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    background-image: url(img/hawa.jpg);
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    height: 20em;
+    padding: 0;
+    margin-left: 1em;
+    filter: opacity(0.5) drop-shadow(0 0 rgb(255, 4, 4));
+  }
+
+  .year {
+    padding-top: 61px;
+    display: flex;
+    align-self: center;
+    color: white;
+    font-size: 38px;
+  }
+
+  .cc {
+    padding-left: 0.5em;
+    color: white;
+    font-size: 1.5em;
+    padding-right: 0.5em;
+  }
+
+  .sec-histo {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
+    background-image: url(img/fem.jpeg);
+    background-repeat: no-repeat;
+    background-size: 100% 100%;
+    height: 20em;
+    padding: 0;
+    margin: 0;
+    filter: opacity(0.5) drop-shadow(0 0 rgb(255, 4, 4));
+  }
+
+  .footer-second2 {
+    margin-top: 5em;
+    width: 100%;
+    display: flex;
+    justify-content: space-around;
+    padding: 3em;
+    background-color: black;
+  }
+
+  .footer-second2 img {
+    width: 12em;
+    height: 3em;
+  }
+
+  #desktop-p {
+    display: none;
+  }
+
+  .footer-second2 p {
+    font-weight: 700;
+    font-size: 10px;
+    line-height: 13px;
+    display: flex;
+    align-items: center;
+    letter-spacing: -0.01em;
+    color: white;
+  }
+
+  footer {
+    min-width: 40em;
+    background-color: black;
+  }
```

Comment:
Please consider using the same `background-color` as used in the design to meet the project requirements.

```
footer {
    background-color: #272A31;
}
```